test/inductor/test_torchinductor.py

```diff
@@ -4890,9 +4900,11 @@ def forward(self, x):
 
         eager_version_counters_after = [
             # TODO: remove the + 1 after https://github.com/pytorch/pytorch/issues/120622 is fixed
-            buffer._version + 1
-            if k in ["m.running_mean", "m.running_var"]
-            else buffer._version
+            (
+                buffer._version + 1
+                if k in ["m.running_mean", "m.running_var"]
+                else buffer._version
+            )
```

Comment:
nit - please remove unrelated formatting changes to a separate pr, it makes the pr harder to review